docs/source/usage.rst

```diff
@@ -257,10 +257,13 @@ collate these separated files into a single file. This is only necessary, and
 supported, for some models.
 
 For most jobs, collation is called automatically. But if you need to manually
-collate the ``K``\ th run, type the following::
+collate the ``K``\ th run (counting from zero), type the following::
```

Comment:
Now with the proliferation of th's I'm concerned this isn't safe in a COVID-19 world ... :) ... also starts being a bit weird awkward. Can we change to something like "collate run output K" and "K-1 restart"?